modin/core/dataframe/pandas/partitioning/partition_manager.py

```diff
@@ -1451,7 +1451,6 @@ def rebalance_partitions(cls, partitions):
                     for obj in partitions[stop]:
                         obj._length_cache = new_last_partition_size
 
-                    partition_size = ideal_partition_size
```

Comment:
Just extra assignment, not directly related to this pull request.

Reply:
Can we revert this change if it's not relevant to the PR?